components/desktop/mate/engrampa/engrampa.p5m

```diff
@@ -23,10 +23,10 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 file path=usr/bin/engrampa
-file path=usr/lib/caja/extensions-2.0/libcaja-engrampa.so
-file path=usr/lib/mate/engrampa-server mode=0555
-file path=usr/lib/mate/engrampa/isoinfo.sh mode=0555
-file path=usr/lib/mate/engrampa/rpm2cpio mode=0555
+file path=usr/lib/$(MACH64)/caja/extensions-2.0/libcaja-engrampa.so
```

Comment:
Note that for this extension  to work caja and engrampa bitness evidently should match